modules/imgcodecs/src/grfmt_webp.cpp

```diff
@@ -331,6 +339,103 @@ bool WebPEncoder::write(const Mat& img, const std::vector<int>& params)
     return size > 0;
 }
 
+bool WebPEncoder::writemulti(const std::vector<Mat>& img_vec, const std::vector<int>& params)
+{
+    Animation animation;
+    animation.frames = img_vec;
+    int timestamp = 0;
+    for (size_t i = 0; i < animation.frames.size(); i++)
+    {
+        animation.timestamps.push_back(timestamp);
+        timestamp += 10;
```

Comment:
I believe this is dangerous to impose a default timestamp. You would end up with two APIs to create a WebP animation: imwritemulti (that imposes a default number of loops, default timestamp, default background) and imwriteanimation with all the features.

For reading, it can be ok: you just want a set of images and that's what you get.

@asmorkalov , what do you think?